website/docs/d/pki_secret_backend_config_est.html.md

```diff
@@ -0,0 +1,70 @@
+---
+layout: "vault"
+page_title: "Vault: vault_pki_secret_backend_config_est data source"
+sidebar_current: "docs-vault-datasource-pki-secret-backend-config-est"
+description: |-
+ Reads the PKI EST configuration from Vault Enterprise. 
+---
+
+# vault\_pki\_secret\_backend\_config\_est
+
+Reads the PKI EST configuration from Vault Enterprise.
+
+~> **Important** All data retrieved from Vault will be
+written in cleartext to state file generated by Terraform, will appear in
+the console output when Terraform runs, and may be included in plan files
+if secrets are interpolated into any resource attributes.
+Protect these artifacts accordingly. See
+[the main provider documentation](../index.html)
+for more details.
+
+## Example Usage
+
+```hcl
+resource "vault_mount" "pki" {
+  path        = "pki"
+  type        = "pki"
+  description = "PKI secret engine mount"
+}
+
+data "vault_pki_secret_backend_config_est" "est_config" {
+  backend     = vault_mount.pki.path
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `namespace` - (Optional) The namespace of the target resource.
+  The value should not contain leading or trailing forward slashes.
+  The `namespace` is always relative to the provider's configured [namespace](/docs/providers/vault/index.html#namespace).
+  *Available only for Vault Enterprise*.
+
+* `backend` - (Required) The path to the PKI secret backend to
+  read the EST configuration from, with no leading or trailing `/`s.
+ 
+## Attributes Reference
+
+* `authenticators` - Lists the mount accessors EST should delegate authentication requests towards (see [below for nested schema](#nestedatt--authenticators))
+ 
+* `default_mount` - Is this mount providing the cluster's default EST mount
```

Comment:
nit: similar comments as resource fields

Reply:
Fixed in https://github.com/hashicorp/terraform-provider-vault/pull/2246/commits/5f5961a4a31806fc4509223e1234830c7a669590